src/main/java/com/linecorp/armeria/client/tracing/TracingRemoteInvoker.java

```diff
@@ -59,7 +60,8 @@ protected TracingRemoteInvoker(RemoteInvoker remoteInvoker, Brave brave) {
                                       Method method, Object[] args) throws Exception {
 
         // create new request adapter to catch generated spanId
-        final InternalClientRequestAdapter requestAdapter = new InternalClientRequestAdapter(method.getName());
+        final InternalClientRequestAdapter requestAdapter = new InternalClientRequestAdapter(
+                Endpoint.create(uri.toString(), 0, 0), method.getName());
```

Comment:
service names should be low cardinality, as they end up in the UI as a drop-down. URI's usually aren't low cardinality.

There's a number of options people use, best is some sort of a platform id for the instance serving the request.